kusd/handler.go

```diff
@@ -628,48 +628,39 @@ func (pm *ProtocolManager) handleMsg(p *peer) error {
 		pm.txpool.AddRemotes(txs)
 
 	case msg.Code == ProposalMsg:
-		// @TODO (rgeraldes) - review flow (we will not need this condition)
-		if !pm.validator.Validating() {
```

Comment:
The flow changed a little bit - We don't need to decode the message if we're not validating. 
With the new flow we decode the message even if it's not necessary.

Reply:
this should be handled in a diff way, but I see why we need the Validating(), adding back